django/db/migrations/questioner.py

```diff
@@ -214,6 +219,30 @@ def ask_not_null_alteration(self, field_name, model_name):
                 return self._ask_default()
         return None
 
+    def ask_not_null_removal(self, field_name, model_name):
+        """Removing a NOT NULL field from a model"""
+        if not self.dry_run:
+            choice = self._choice_input(
+                f"It is impossible to remove a non-nullable field '{field_name}' "
+                f"from '{model_name}' without a default; "
+                f"This is because the database needs something to populate existing "
+                f"rows when migrating backwards.\n"
```

Comment:
I'm going to be direct here to not waste time 🙂

I originally commented about this prompt:

> Also "can't do that" is misleading. Some people may be quite happy to proceed without a valid reverse operation.

You've gone from "We can't do that" to "It is impossible…". No, it is not impossible. You're still too focused on this idea that Django must complain and raise errors. It shouldn't; Django should only warn and suggest or hint that the user may wish to add a value to make a valid backwards migration.

Reply:
Agree 👍 , would ``risky`` in place of ``impossible`` sound ok?